packages/test/test-end-to-end-tests/src/test/gc/gcTestSummaryUtils.ts

```diff
@@ -29,10 +36,33 @@ export function getGCStateFromSummary(summary: ISummaryTree): IGarbageCollection
         }
 
         const gcBlob = rootGCTree.tree[key];
-        assert(gcBlob?.type === SummaryType.Blob, `GC blob not available`);
+        assert(gcBlob !== undefined, "GC state not available");
+        assert(gcBlob.type === SummaryType.Blob, "GC state is not a blob");
         const gcState = JSON.parse(gcBlob.content as string) as IGarbageCollectionState;
         // Merge the GC state of this blob into the root GC state.
         rootGCState = concatGarbageCollectionStates(rootGCState, gcState);
     }
     return rootGCState;
 }
+
+/**
+ * Returns the tombstone data from the GC tree in the summary.
+ * Note that it assumes that the tombstone data in the GC tree is a summary blob and not summary handle.
+ * @param summaryTree - The summary tree that contains the GC summary.
+ * @returns The tombstone data if it exists, undefined otherwise.
+ */
+export function getGCTombstoneStateFromSummary(summaryTree: ISummaryTree): string[] | undefined {
```

Comment:
Looks like this is only used in the tests, is there a reason why this is here?

Reply:
This file is in test folder. Do you mean its only used in 1 test and it could be in that file itself? If so, yes it could be. I kept it here just because its similar to the function above. These two functions could be combined too (which is what I initially did) but then a lot of tests had to be updated making the PR huge! Maybe I can do that as a follow up. Let me know what you think.

Reply:
Some how I thought this was not in the test folder